setup.py

```diff
@@ -69,7 +69,7 @@
         "scipy~=1.2",
         "tables~=3.4",
         "tqdm>=4.32",
-        "traitlets>=4.1,<5.0",
+        "traitlets~=5.0,>=5.0.5",
```

Comment:
Is this not identical to `"traitlets~=5.0.5"`?

Reply:
No, since `~=5.0.5` would fix us to `5.0.x`